Editor/Windows/CinemachineSettings.cs

```diff
@@ -329,7 +329,13 @@ private static void OnGUI()
                 new GUIContent("Storyboard Global Mute", "If checked, all storyboards are globally muted."), 
                 CinemachineStoryboardMute.Enabled);
 #endif
-
+            // Editable Game Window Guides
+            CinemachineScreenComposerGuidesGlobalEditable.Enabled = EditorGUILayout.Toggle(
+                new GUIContent("Editable Game Window Guides", "If checked, game window guides are editable in play " +
```

Comment:
For me, it's cut off.  Please fix.

![image](https://user-images.githubusercontent.com/6424658/110128716-1e7c6500-7d95-11eb-8741-f4c218f2ad3e.png)